atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -102,6 +103,7 @@
     Map<InetSocketAddress, CassandraClientPoolingContainer> currentPools = Maps.newConcurrentMap();
     final CassandraKeyValueServiceConfig config;
     final ScheduledThreadPoolExecutor refreshDaemon;
+    private final Optional<MetricRegistry> metricRegistry;
```

Comment:
Probably need to hook in a global MetricRegistry per https://github.com/palantir/atlasdb/pull/1356